dmithamo1/task-manager-api
# #1 Configure Project's Schema

#### What does this PR do?
This PR configures the project ahead of the commencement of developemnt
#### Description of Task to be completed
- set up an appropriate folder structure
- install base requirements
#### How should this be manually tested?
1. Clone this repo
2. Run the following commands in Terminal
```
cd tasks_api
pip install -r requirements.txt
python manage.py runserver
```
3. Navigate to `http://127.0.0.1:8000/` on your browser
4. You should see a screen similar to the one shown below (Screenshots) if all went well
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/45358371/55823118-bc177d00-5b09-11e9-974f-f08ccd44b06b.png)

#### Checklist
